src/plugins/mmapstorage/mmapstorage.c

```diff
@@ -966,6 +966,10 @@ static void updatePointers (MmapMetaData * mmapMetaData, char * dest)
 		{
 			key->meta = (KeySet *) ((char *) (key->meta) + destInt);
 		}
+		if (key->ukey)
+		{
+			key->ukey = key->ukey + destInt;
+		}
```

Comment:
Sounds like the right direction, but there are other places we need to change. We will also need to copy this data into the `mmap` region.